src/nft/hooks/index.ts

```diff
@@ -0,0 +1,21 @@
+export * from './useCart'
```

Comment:
Our current style mostly avoids barrel files like this.
Given the large migration, I think it is fine, but it should be avoided in the future.

Reply:
When we start making long term architecture discussions I would be open to restarting this discussion as we have found barrel files to be very help organizationally